### DISCUSSION TITLE
하드디스크 할당 관련 질문

### DISCUSSION THREAD

**Discussion by sun3702** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#80] [2024년도 기타 질문]
노트북 용량이 부족하여 최대할당공간이 30GB가 안되는데 혹시 해당 용량으로 진행해도 과제 수행 가능한지 문의 드립니다.

![image](https://github.com/user-attachments/assets/63b14f8d-e432-4b5b-8343-3914f93f761c)

**Comment by HyeongKeunHong** [comment]
해당 용량으로는 과제수행이 어렵습니다.

만일 디스크상에 충분한 용량이 있음에도 불구하고 축소할 공간이 적게 나타난다면, 아래의 링크를 참고하셔서 진행하시면 용량 문제가 해결될 것입니다.

https://archive2497.tistory.com/entry/%EC%9D%B4%EB%8F%99%ED%95%A0-%EC%88%98-%EC%97%86%EB%8A%94-%ED%8C%8C%EC%9D%BC%EC%9D%B4-%EC%9E%88%EB%8A%94-%EC%A7%80%EC%A0%90%EC%9D%84-%EB%B2%97%EC%96%B4%EB%82%98-%EB%B3%BC%EB%A5%A8%EC%9D%84-%EC%B6%95%EC%86%8C%ED%95%A0-%EC%88%98-%EC%97%86%EC%8A%B5%EB%8B%88%EB%8B%A4-%ED%95%B4%EA%B2%B0%EB%B0%A9%EB%B2%95-%EC%A0%95%EB%A6%AC

**Reply by sun3702** [reply]
해결되었습니다! 감사합니다!
